README.md

```diff
@@ -616,3 +616,36 @@ npm run updateLocalEnv
 ```
 
 This will replace your `.env.local` file with the one that will be used in prod (simply taking `.env.template + .env.SECRET_CONFIG`).
+
+### Populate database
+
+You can populate the database using faker data using the `populate` script:
+
+```bash
+npm run populate <flags here>
+```
+
+At least one flag must be specified, the following flags are available:
+
+- `reset` - resets the database
+- `users` - populates the users table
+- `settings` - populates the settings table for existing users
+- `assistants` - populates the assistants table for existing users
+- `conversations` - populates the conversations table for existing users
+
+> [!WARNING]  
+> The `MONGODB_URL` used for this script will be fetched from `.env.local`. Make sure it's correct! The command runs directly on the database.
```

Comment:
this section should be placed right after `### Populate database`

Reply:
Agreed, done!